recipes/qbs/all/conanfile.py

```diff
@@ -0,0 +1,83 @@
+from conans import ConanFile, tools
+import os
+
+
+class QbsConan(ConanFile):
+    name = "qbs"
+    description = "The Qbs build system"
+    topics = ("conan", "qbs", "build", "automation")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://qbs.io"
+    license = "LGPL-2.1-only", "LGPL-3.0-only", "Nokia-Qt-exception-1.1"
+    settings = {
+        "arch_build": ["x86_64"],
+        "os_build": ["Linux", "Macos", "Windows"]
+    }
+
+    def source(self):
+        pass
+
+    def build(self):
+        data = dict(**self.conan_data["sources"][self.version][str(self.settings.os_build)])
+        tools.download(**data, filename="creator.7z")
+        self.run("cmake -E tar x creator.7z")
+
+    def package(self):
+        if self.settings.os_build == "Linux":
+            self.copy("bin/qbs*", symlinks=True)
+            self.copy("bin/qt.conf", symlinks=True)
+            self.copy("lib/qtcreator/libqbs*", symlinks=True)
+            self.copy("lib/qtcreator/plugins/qbs/*", symlinks=True)
+            self.copy("lib/Qt/lib/libic*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5Core*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5DBus*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5Gui*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5Network*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5PrintSupport*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5Script*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5Widgets*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5Xml*", symlinks=True)
+            self.copy("lib/Qt/lib/libQt5XcbQpa*", symlinks=True)
+            self.copy("lib/Qt/plugins/platforms/libqxcb*", symlinks=True)
+            self.copy("lib/Qt/plugins/platformthemes/libqgtk3*", symlinks=True)
+            self.copy("libexec/qtcreator/qbs/*", symlinks=True)
+            self.copy("share/qtcreator/qbs/*", symlinks=True)
```

Comment:
The hooks will complain because `libexec` and `share` are not whitelisted.
Can these be moved?

Same goes for Macos.

Reply:
I don't think they can be moved. The structure is defined at compile time.

Reply:
OK, I found a FAQ entry which says that all content of binary tools should be placed in bin. Done.